docs/docs/reference/release-notes/v2.30/index.md

```diff
@@ -0,0 +1,189 @@
+---
+date: "2021-01-05"
+version: "2.30.0"
+---
+
+# [v2.30](https://github.com/gatsbyjs/gatsby/compare/gatsby@2.30.0-next.0...gatsby@2.30.0) (January 2021 #1)
+
+---
+
+Welcome to `gatsby@2.30.0` release (January 2021 #1)
+
+Key highlights of this release:
+
+- [Query on Demand and Lazy Images: Generally available](#query-on-demand-and-lazy-images-generally-available) - improves `gatsby develop` bootup time
+- [Server Side Rendering (SSR) in development](#server-side-rendering-ssr-in-development) — helps you find and fix many build errors in development. We're starting a partial release of this feature to 5% of users.
+- [Image transformations up to 30% faster](#gatsby-plugin-sharp2120)
+- [gatsby-plugin-sass v3](#gatsby-plugin-sass300) - use the latest `sass-loader` and `Dart Sass` by default
```

Comment:
Nit: table of content order doesn't match content order (in content we have sass before sharp) ;)